conf/modules.config

```diff
@@ -106,6 +108,8 @@ process {
     }
 
     withName: DRAM_DISTILL {
+        cpus   = 1
+        memory = { 24.GB * task.attempt }
```

Comment:
I guess the memory required depends on the size of the input, right?

Reply:
It depends on the number of samples and the complexity of each sample in terms of the number of different species found. That impacts the number of annotated functions